pkg/sql/schema_changer.go

```diff
@@ -601,6 +600,12 @@ func (sc *SchemaChanger) exec(ctx context.Context) error {
 					return err
 				}
 			}
+		case sqlbase.DatabaseDescriptor:
```

Comment:
you could just add `sqlbase.DatabaseDescriptor` to the case above

Reply:
Done.